tracker/storage.go

```diff
@@ -14,43 +14,10 @@
 package tracker
 
 import (
-	"database/sql"
 	"github.com/hashicorp/go-memdb"
-	_ "github.com/mattn/go-sqlite3"
-	"log"
 	"sync/atomic"
 )
 
-const (
-	DB_DRIVER       = "sqlite3"
-	DB_TABLE_NAME   = "events"
-	DB_COLUMN_ID    = "id"
-	DB_COLUMN_EVENT = "event"
-)
-
-type Storage interface {
-	AddEventRow(payload Payload) bool
-	DeleteAllEventRows() int64
-	DeleteEventRows(ids []int) int64
-	GetAllEventRows() []EventRow
-	GetEventRowsWithinRange(eventRange int) []EventRow
-}
-
-type RawEventRow struct {
-	id    int
-	event []byte
-}
-
-type RawEventRowUint struct {
-	id    uint
-	event []byte
-}
-
-type EventRow struct {
-	id    int
-	event Payload
-}
-
 // --- Memory Storage Implementation
 
 type StorageMemory struct {
```

Comment:
I'd prefer to call this file `storage_memory.go` or something else that reflects this is an implementation and rename `storage_interface.go` to`storage.go`